website/public/index.html

```diff
@@ -96,9 +92,27 @@ <h1>Learn sbpy</h1>
     </section>
 
     <section class="whatsnext">
-      <h1>Report bugs and Contribute</h1>
-      <p>Please report what you believe is a bug or a mistake using the github issue tracker. If you have an idea for
-        how to improve sbpy, or would like to contribute code directly, please check out the contribution guidelines.
+      <h1>Get help</h1>
+      <p>Questions about sbpy or how to use sbpy modules can be posted at <a
+          href="https://github.com/NASA-Planetary-Science/sbpy/discussions">sbpy
+          Discussions</a>, hosted on GitHub. Help about using astropy and affiliated
+        packages (including sbpy) can also be found on <a href="https://www.astropy.org/help.html">Astropy's help
+          page</a>. You can
+        also find sbpy developers on the Astropy slack workspace in the #sbpy
+        channel (see <a href="https://www.astropy.org/help.html">Astropy's help
+          page</a> for connection details).
+      </p>
+      <a class="button" href="https://github.com/NASA-Planetary-Science/sbpy/discussions" target="_blank">sbpy
+        Discussions</a>
+      <a class="button" href="https://www.astropy.org/help.html" target="_blank">Astropy help</a>
+    </section>
+
+    <section class="whatsnext">
+      <h1>Report bugs and contribute</h1>
+      <p>Please report what you believe is a bug or a mistake using <a
+          href="https://github.com/NASA-Planetary-Science/sbpy/issues">sbpy's issue
+          tracker</a>. If you have an idea for how to improve sbpy, or would like to
+        contribute code directly, please check out the contribution guidelines.
```

Comment:
add a link to the development page https://sbpy.readthedocs.io/en/latest/development/index.html for "contribution guidelines"?

Reply:
Done.